echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/EventContent.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2015 Netflix, Inc.
```

Comment:
Was this added by you or translated from a netflix groovy class?

Reply:
It was added by me to "hide" the ugly Map navigations in the Event in Java. Maybe you have some suggestion on how to do it nicer?

I'm fixing the license